### PR TITLE
feat(widget-add-modal): add DashboardAddWidgetModal

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
스크린샷 2022-12-08 오후 5 26 50

- just markup
- **Most of codes are copied from cost-explorer** 
- sorry for big commit 😅

### CODE REVIEW

File: src/services/dashboards/dashboard-customize/modules/DashboardAddWidgetModal.vue [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        $tab-bar: 3rem;
+
+        /* custom design-system component - p-button-modal */
+        :deep(.modal-body) {
+            @apply overflow-hidden;
+            .tab-pane {
+                height: calc(100vh - $modal-hedaer - $modal-padding - $modal-footer - $tab-bar);
+            }
+        }
+    }
```
Comment (WANZARGEN) [inline src/services/dashboards/dashboard-customize/modules/DashboardAddWidgetModal.vue:86]: question: oh.. Is there no `xl` size in button modal component?

**Review by minseolee — APPROVED** [review]
LGTM! I did not check deeply

File: src/services/dashboards/dashboard-customize/modules/DashboardAddWidgetModal.vue [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+export default defineComponent<Props>({
+    name: 'DashboardAddWidgetModal',
+    components: {
+        DashboardDefaultWidgetTab,
+        PTab,
+        PButtonModal,
+    },
+    props: {
+        visible: {
+            type: Boolean,
```
Comment (minseolee) [inline src/services/dashboards/dashboard-customize/modules/DashboardAddWidgetModal.vue:46]: there is no `default` value. Is it fine??

File: src/services/dashboards/dashboard-customize/modules/DashboardDefaultWidgetTab.vue [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+                    :class="{'selected' : selectedWidgetConfigId === widget.widget_config_id}"
+                    @click="selectWidget(widget.widget_config_id)"
+                >
+                    <div class="card-header">
+                        {{ widget.title }}
+                    </div>
+                    <div class="card-content">
+                        <img class="card-image"
+                             :src="widget.description?.preview_image"
+                        >
```
Comment (WANZARGEN) [inline src/services/dashboards/dashboard-customize/modules/DashboardDefaultWidgetTab.vue:21]: suggestion: I recommend to use `p-lazy-img` component

**Review by WANZARGEN — APPROVED** [review]
Nice clean code :)
I left some comments!

File: src/services/dashboards/dashboard-customize/modules/DashboardAddWidgetModal.vue (hunk 2 above)
Comment (yuda110) [reply src/services/dashboards/dashboard-customize/modules/DashboardAddWidgetModal.vue:46]: Oh I missed it! thanks!

**Pull request merged**